app/liquid_tags/codepen_tag.rb

```diff
@@ -31,7 +31,7 @@ def parse_options(input)
     _, *options = stripped_link.split
 
     # Validation
-    validated_options = options.map { |option| valid_option(option) }.reject(&:nil?)
+    validated_options = options.filter_map { |option| valid_option(option) }
```

Comment:
At first it suggested changing `reject(&:nil?)` to `compact`, then it suggest changing `map` + `compact` to `filter_map`.